### ISSUE TITLE
Настроить срок хранения логов

### ISSUE THREAD

**Issue by dima117** [dima117/thinking-home#7]
Чтобы старые логи автоматически удалялись через заданное количество дней и не накапливались.

**Comment by om2804** [comment]
Логирование выполняется через NLog. Данная фича решается настройкой конфига https://github.com/nlog/nlog/wiki/File-Target

**Comment by dima117** [comment]
Да, это и имелось в виду. Речь шла о настройке конфига по умолчанию, который копируется на компьютер при установке.

Кстати, в [новой версии](https://github.com/thinking-home/system) (на платформе .NET Core) вместо NLog используется [Serilog](https://serilog.net), там по умолчанию хранится 31 день.
